dist/js/game.js

```diff
@@ -1,5 +1,7 @@
 
-let movement_speed = 5; 
+let movement_speed = 2; 
+let su = new SpriteUtilities(PIXI);
```

Comment:
```suggestion
const su = new SpriteUtilities(PIXI);
```
Inte helt säker dock

Reply:
Kanske ge den ett mer beskrivande namn också